### PR TITLE
doc: update nrfxlib revision for mpsl figures update

### PR DESCRIPTION
**Pull request by Avalei**
For https://github.com/nrfconnect/sdk-nrfxlib/pull/593

Signed-off-by: Deidre Casey <deidre.casey@nordicsemi.no>

### CODE REVIEW

**Comment by Avalei** [comment]
This PR was only used to generate a build preview for nrfconnect/sdk-nrfxlib#593 and is no longer needed.

**Comment by nvlsianpu** [comment]
labeled `CI-disable` as nrfxlib update contains only documentation patch.

File: west.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: fd79b800bddf63b0a212a70007a6b612226e134c
+      revision: pull/593/head
```
Comment (nvlsianpu) [inline west.yml:107]: SHA: b0a356db59463e7f16f59fea5ca9a7f73c1b6ae1

**Review by nvlsianpu — CHANGES_REQUESTED** [review]

File: west.yml (hunk 1 above)
Comment (Avalei) [reply west.yml]: Thanks, just updated this with the correct SHA.

**Review by nvlsianpu — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**